Add draw.io library (#1881)
nf-core/website: src/content/docs/contributing/design_guidelines.md

| <img title="Metromap style pipeline workflow components" src="/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.svg" width="200"> | Components for a metro-map style pipeline workflow | [SVG](/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.svg) <br> [PDF](/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.pdf) | James A. Fellows Yates & nf-core; under a [CC0](https://creativecommons.org/publicdomain/zero/1.0/?ref=chooser-v1) license (public domain) |

### Use draw.io

> The website / app [draw.io](https://app.diagrams.net/) help you create, render and export different diagram figures including metro-map.

Reviewer: ```suggestion
The web app [draw.io](https://app.diagrams.net/) helps you create, render and export different diagrams including metro-maps.
```

> For even more convenience you can use the following library [NF-Core xml item library](https://github.com/LouisLeNezet/phaseimpute/blob/dev/docs/NfCore_library.xml).

Reviewer: ```suggestion
For even more convenience you can use the following asset library [NF-Core xml item library](https://github.com/LouisLeNezet/phaseimpute/blob/dev/docs/NfCore_library.xml). It contains all of the components from the cheatsheet above.
```

To add it to your diagram use "File / Open library from / URL..." and paste:

> ```

Reviewer: ```suggestion
```plaintext
```
with this we should get a copy-button
